dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/pages/project/workflow/WorkflowForm.java

```diff
@@ -60,26 +70,50 @@ public WorkflowForm(WebDriver driver) {
 
         final JavascriptExecutor js = (JavascriptExecutor) driver;
         final String dragAndDrop = String.join("\n",
-            Resources.readLines(Resources.getResource("dragAndDrop.js"), StandardCharsets.UTF_8));
+                Resources.readLines(Resources.getResource("dragAndDrop.js"), StandardCharsets.UTF_8));
         js.executeScript(dragAndDrop, task, canvas);
 
         switch (type) {
             case SHELL:
                 return (T) new ShellTaskForm(this);
             case SUB_PROCESS:
                 return (T) new SubWorkflowTaskForm(this);
+            case SWITCH:
+                return (T) new SwitchTaskForm(this);
         }
         throw new UnsupportedOperationException("Unknown task type");
     }
 
+    public WebElement getTask(String taskName) {
+        List<WebElement> tasks = new WebDriverWait(driver, 10)
+                .until(ExpectedConditions.visibilityOfAllElementsLocatedBy(By.cssSelector("svg > g > g[class^='x6-graph-svg-stage'] > g[data-shape^='dag-task']")));
```

Comment:
Hi @caishunfeng , can we get rid of this kind of selector by adding a class or id to the svg node?

Reply:
I asked @songjianet, the task svg node id is randomly generated, and class selector is like css selector I used, so maybe we can use it first, and then optimize it later if need.